samples/GenMintEntitlement.js

```diff
@@ -0,0 +1,75 @@
+const { ElvClient } = require("@eluvio/elv-client-js");
+const crypto = require("crypto");
+const { ethers } = require('ethers');
+const bs58 = require("bs58");
+
+let client;
+
+/* Sample configuration */
+let tenant = "iten4TXq2en3qtu3JREnE5tSLRf9zLod"; // paladin
+let marketplaceObjectId = "iq__2dXeKyUVhpcsd1RM6xaC1jdeZpyr"; // A Place for Goats
+let sku = "C9Zct19CoEAZYWug9tyavX"; // Goat Pack One
+let amount = 1;
+let nonce = "nonce_6f9f53ecc09a7e223cf7d47f";
+let purchaseId = "pid_e852572c6e84626892da049a";
+
+/**
+ * Generate a mint entitlement
+ *
+ * @param {string} tenant - tenant ID in 'iten' format
+ * @param {string} marketplaceObjectId - marketplace object ID in 'iq__' format
+ * @param {string} sku - SKU of the item
+ * @param {number} amount - number of items of that SKU
+ * @returns {Promise<Object>} - the entitlement JSON and signature
+ */
+const Entitlement = async({tenant, marketplaceObjectId, sku, amount, nonce, purchaseId}) => {
+  const message = {
+    tenant_id: tenant,
+    marketplace_id: marketplaceObjectId,
+    items: [ { sku: sku, amount: amount } ],
+    nonce: nonce,
+    purchase_id: purchaseId,
+  };
+
```

Comment:
should it be `personal_sign` message?

```
message = typeof message === "object" ? JSON.stringify(message) : message;
message = Ethers.utils.keccak256(Buffer.from(`\x19Ethereum Signed Message:\n${message.length}${message}`, "utf-8"));
```

Reply:
nah we want it to use the existing client function with as little dressing as possible.  if the client lib wants to do that internally in some client.SignZXC(), that's fine tho.